src/frontend/components/Chat/SharedChatComponents/ChatInputBar/index.spec.tsx

```diff
@@ -13,52 +12,39 @@ jest.mock('utils/window', () => ({
   },
 }));
 
+const mockClickOnJoinButton = jest.fn();
+
 describe('<ChatInputBar />', () => {
   beforeEach(() => {
     jest.resetAllMocks();
   });
 
-  it("doesn't send message if the send button is clicked when input is empty", async () => {
-    render(wrapInIntlProvider(<ChatInputBar />));
-    const chatInputText = screen.getByRole('textbox');
-    const chatSendButton = screen.getByRole('button');
+  it("renders 'join chat' button and clicked on it.", async () => {
```

Comment:
```suggestion
  it("renders 'join chat' button and click on it.", async () => {
```

Reply:
The component ChatInputBar doesn't exist anymore.